### PR TITLE
Movie detail screen

### PR DESCRIPTION
**Pull request by knowpage**

### CODE REVIEW

File: lib/presentation/journey/home/movie_detail/movie_detail_widget.dart [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+                                ),
+                                const Icon(
+                                  Icons.keyboard_arrow_down_rounded,
+                                  color: Color(AppColors.white),
+                                ),
+                              ],
+                            ),
+                          ),
+                          const Align(
+                            child: TicketButtonWidget(
```
Comment (ntlam6) [inline lib/presentation/journey/home/movie_detail/movie_detail_widget.dart:136]: what does Align do here

File: lib/presentation/journey/home/movie_detail/movie_detail_widget.dart [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+                            height: 8,
+                          ),
+                          Text(
+                            'Cast',
+                            style: AppTextStyle.movieRateTile,
+                          ),
+                          SizedBox(
+                            child: ListView.builder(
+                              itemBuilder: (_, index) {
+                                return Row(
```
Comment (ntlam6) [inline lib/presentation/journey/home/movie_detail/movie_detail_widget.dart:185]: Row in unnecessary

File: lib/presentation/journey/home/movie_detail/movie_detail_widget.dart [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+                            height: 8,
+                          ),
+                          Text(
+                            'Screenshots',
+                            style: AppTextStyle.movieRateTile,
+                          ),
+                          SizedBox(
+                            child: ListView.builder(
+                              itemBuilder: (_, index) {
+                                return Row(
```
Comment (ntlam6) [inline lib/presentation/journey/home/movie_detail/movie_detail_widget.dart:150]: Row in unnecessary

File: lib/presentation/journey/home/movie_detail/movie_detail_widget.dart [hunk 4]
```diff
@@ -0,0 +1,220 @@
...
+                        isReadMore = state.isReadMore;
+                        if (state.isReadMore) {
+                          lines = 1;
+                          textButton = "Read less";
+                        } else {
+                          lines = 3;
+                          textButton = "Read more";
+                        }
+                      }
+                      return ListView(
```
Comment (ntlam6) [inline lib/presentation/journey/home/movie_detail/movie_detail_widget.dart:95]: Why use ListView here

File: lib/presentation/journey/home/movie_detail/widget/movie_detail_slider_widget.dart [hunk 5]
```diff
@@ -0,0 +1,61 @@
+import 'package:flutter/material.dart';
+import 'package:movie_app/presentation/journey/home/movie_detail/widget/carosel_slider/carousel_options.dart';
+import 'package:movie_app/presentation/journey/home/movie_detail/widget/carosel_slider/carousel_slider.dart';
+
+class DemoItem extends StatelessWidget {
```
Comment (ntlam6) [inline lib/presentation/journey/home/movie_detail/widget/movie_detail_slider_widget.dart:5]: why no Widget in name

File: lib/presentation/journey/home/movie_detail/widget/movie_detail_slider_widget.dart [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+    return ListTile(
+      title: Text(title),
+      onTap: () {
+        Navigator.pushNamed(context, route);
+      },
+    );
+  }
+}
+
+class MultipleItemDemo extends StatelessWidget {
```
Comment (ntlam6) [inline lib/presentation/journey/home/movie_detail/widget/movie_detail_slider_widget.dart:21]: Remove if not used

File: lib/presentation/my_app.dart [hunk 7]
```diff
@@ -1,16 +1,30 @@
 import 'package:flutter/material.dart';
+import 'package:flutter_localizations/flutter_localizations.dart';
+import 'package:movie_app/common/constant/colors.dart';
 import 'journey/home/home_widget.dart';
+// import 'package:flutter_gen/gen_l10n/app_localizations.dart';
```
Comment (ntlam6) [inline lib/presentation/my_app.dart:5]: remove if not needed

File: lib/presentation/journey/home/movie_detail/movie_detail_widget.dart (hunk 1 above)
Comment (knowpage) [reply lib/presentation/journey/home/movie_detail/movie_detail_widget.dart:136]: deleted

File: lib/presentation/journey/home/movie_detail/movie_detail_widget.dart (hunk 2 above)
Comment (knowpage) [reply lib/presentation/journey/home/movie_detail/movie_detail_widget.dart:185]: deleted

File: lib/presentation/journey/home/movie_detail/movie_detail_widget.dart (hunk 3 above)
Comment (knowpage) [reply lib/presentation/journey/home/movie_detail/movie_detail_widget.dart:150]: deleted

File: lib/presentation/journey/home/movie_detail/movie_detail_widget.dart (hunk 4 above)
Comment (knowpage) [reply lib/presentation/journey/home/movie_detail/movie_detail_widget.dart:95]: I don't know

File: lib/presentation/journey/home/movie_detail/widget/movie_detail_slider_widget.dart (hunk 5 above)
Comment (knowpage) [reply lib/presentation/journey/home/movie_detail/widget/movie_detail_slider_widget.dart:5]: deleted

File: lib/presentation/journey/home/movie_detail/widget/movie_detail_slider_widget.dart (hunk 6 above)
Comment (knowpage) [reply lib/presentation/journey/home/movie_detail/widget/movie_detail_slider_widget.dart:21]: deleted

File: lib/presentation/my_app.dart (hunk 7 above)
Comment (knowpage) [reply lib/presentation/my_app.dart:5]: okay

**Pull request merged**